### DISCUSSION TITLE
Unable to run NextJs Application.

### DISCUSSION THREAD

**Discussion by AestheticAaditya0203** [vercel/next.js#50773] [Help]
### Summary

I am facing some error while running nextjs application, surely related to some package of dependencies, tried various solutions but was unable to sort it out. Can you guys please help me with it.
 Error: 
     dyld: lazy symbol binding failed: Symbol not found: _pthread_jit_write_protect_supported_np
  Referenced from: /Users/appiness/Desktop/QuickbytesProject/QuickbytesUser/quickbytes-fed/node_modules/sharp/build/Release/../.././vendor/8.13.3/darwin-x64/lib/libvips-cpp.42.dylib
  Expected in: /usr/lib/libSystem.B.dylib

dyld: Symbol not found: _pthread_jit_write_protect_supported_np
  Referenced from: /Users/appiness/Desktop/QuickbytesProject/QuickbytesUser/quickbytes-fed/node_modules/sharp/build/Release/../.././vendor/8.13.3/darwin-x64/lib/libvips-cpp.42.dylib
  Expected in: /usr/lib/libSystem.B.dylib

**Comment by SamKomesarook** [comment]
Are you able to share the repo?
